services/QuillLMS/app/controllers/auth/clever_controller.rb

```diff
@@ -16,8 +16,8 @@ def clever
       @auth_hash ||= request.env['omniauth.auth']
     end
 
-    private def district_success(data, redirect=nil)
-      render status: 200, nothing: true # Don't bother rendering anything.
+    private def district_success(data, redirect)
```

Comment:
can we remove both params here, so that the method signature matches the one in controllers/clever_controller?